docs/nuget-org/Deprecate-packages.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deprecating packages on nuget.org
+description: Detailed description on the process of deprecating packages and how the clients shows this information
+author: anangaur
+ms.author: anangaur
+ms.date: 09/23/2019
+ms.topic: conceptual
+ms.reviewer: karann-msft
+---
+
+# Deprecating packages
+
+As a package publisher, if you no longer want to maintain a package or would like to encourage your package's consumers to move to another package for various reasons, you can now deprecate the package. 
+
+Package deprecation is different than **unlisting** your package as explained below:
+* **Unlisting** a package prevents its discovery because it is hidden in search results. This prevents new installs of the package.
```

Comment:
> This prevents new installs of the package.

I don't know if I'd keep that. I don't want it to suggest that there's no way to install that specific version of the package.

Reply:
I agree. I think this bullet point would be better without the last sentence